.ci/metrics/metrics.py

```diff
@@ -164,16 +230,19 @@ def main():
     # Enter the main loop. Every five minutes we wake up and dump metrics for
     # the relevant jobs.
     while True:
-        current_metrics = get_metrics(github_repo, workflows_to_track)
+        current_metrics = get_per_workflow_metrics(github_repo, workflows_to_track)
+        current_metrics += get_sampled_workflow_metrics(github_repo)
         if len(current_metrics) == 0:
-            print("No metrics found to upload.", file=sys.stderr)
-            continue
+            print("No metrics found to upload.", file=sys.stdout)
 
         upload_metrics(current_metrics, grafana_metrics_userid, grafana_api_key)
-        print(f"Uploaded {len(current_metrics)} metrics", file=sys.stderr)
+        print(f"Uploaded {len(current_metrics)} metrics", file=sys.stdout)
```

Comment:
Is there a reason you've changed this to `stdout`? I think `stderr makes more sense here given it's for debug logging rather than any actual application output.

Reply:
This is not an error, just a log. GCP currently reports those as severe errors since we don't use the GCP logging library.
My hope is if we output to stdout (as a non-error should), GCP will classify this as `INFO`

Reply:
Let's keep everything on `stderr` for now and revisit later on.